views/web_views.py

```diff
@@ -606,6 +606,8 @@ def assets_all_styles():
         concat_asset_files(
             [
                 "static/css/vendor-bootstrap-4.0.0-beta2.css",
+                "static/css/vendor-owl.theme.default.min.css",
+                "static/css/vendor-owl.carousel.min.css",
```

Comment:
I'm not sure what's our take on adding thrid-party deps to the website. But think it's fine. 
Tagging @joshfraser @sparrowDom for their opinion.

Reply:
If it is working fine I don't have a problem with it